### PR TITLE
maps: Fix NAT map retrieval with IPv4 disabled and IPv6 enabled

### PR DESCRIPTION
**Pull request by tgraf**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/8413)

### CODE REVIEW

**Review by joestringer — APPROVED** [review]

**Comment by tgraf** [comment]
test-me-please

File: pkg/maps/nat/nat.go [hunk 1]
```diff
@@ -240,18 +240,13 @@ func (m *Map) DeleteMapping(key tuple.TupleKey) error {
 	return deleteMapping6(m, key.(*tuple.TupleKey6Global))
 }
 
-func maps(ipv4, ipv6 bool) []*Map {
-	result := make([]*Map, 0, mapCount)
+// GlobalMaps returns all global NAT maps.
+func GlobalMaps(ipv4, ipv6 bool) (ipv4Map, ipv6Map *Map) {
```
Comment (borkmann) [inline pkg/maps/nat/nat.go:244]: Looks good, I presume this works fine with the way we handle commands: https://github.com/cilium/cilium/blob/319ccde9f19deaa435dd6d29a84ee13cfcea90fa/cilium/cmd/bpf_nat_flush.go#L42

Comment (borkmann) [reply pkg/maps/nat/nat.go:244]: Ok, judging from travis, looks like this needs to be fixed for the cli.

Comment (tgraf) [reply pkg/maps/nat/nat.go:244]: Yep, fixed in latest force push

**Comment by joestringer** [comment]
test-me-please

**Pull request merged**